eclass/kernel-install.eclass

```diff
@@ -38,13 +52,117 @@ esac
 
 inherit dist-kernel-utils mount-boot toolchain-funcs
 
+LICENSE="GPL-2"
+if [[ ${KERNEL_IUSE_GENERIC_UKI} ]]; then
+	INITRD_PACKAGES=(
+		"app-arch/gzip GPL-3+"
```

Comment:
Here's another idea:

```suggestion
	declare -A INITRD_PACKAGES=(
		[app-arch/gzip]="GPL-3+"
```

Reply:
This syntax is new for me, but it does the job and does indeed look a bit nicer.

I do have to use a helper variable `INITRD_PACKAGES_BDEPEND="${!INITRD_PACKAGES[@]}"`. Somehow the variable INITRD_PACKAGES` can't be accessed in `kernel-build.eclass` when it is declared in this way, I tried `export` but this did not help.

Reply:
Try `declare -g`.

Reply:
> Try `declare -g`.

Yes! That does the trick!